RealmSwift/Object.swift

```diff
@@ -277,9 +277,15 @@ open class Object: RLMObjectBase, ThreadConfined {
 
      - parameter object: The object to compare the receiver to.
      */
-    open override func isEqual(_ object: Any?) -> Bool {
+#if os(macOS)
```

Comment:
Hmm. Is this necessary due to the weird scripting category that adds methods to NSObject on Mac?

I wonder if we should give this a name that more clearly distinguishes it from `isEqual(_:)`?